lib/cc/analyzer/container.rb

```diff
@@ -67,6 +67,12 @@ def run(options = [])
         end
       end
 
+      def stop
+        @on_output = ->(*) { }
+        Process.kill("TERM", @pid)
```

Comment:
Looks like if `@pid` were `nil` we would get a `TypeError: no implicit conversion from nil to integer` but I don't think that's possible. `Container#stop` is only called in response to output emitted from the running engine which would have been assigned a PID. If the process terminates before we send a `TERM` command, we'll get an `Errno::ESRCH` exception, which we're already handling with a no-op rescue.